spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe SearchController do
+  describe '#index' do
+    context 'when requesting html view' do
+      it 'responses with oki' do
+        get :index
+
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when requested with another format' do
+      it 'responses with a 404' do
+        get :index, format: :rss
+
```

Comment:
Please add a mock to test that a search didn't run, maybe that `Search.new` or `.search_for_user` wasn't called.